pkg/events/grpc/grpc.go

```diff
@@ -80,18 +81,14 @@ type EventsServer struct {
 	filter events.IdentifierFilter
 }
 
-var (
-	evtStreamStart = events.Define("events.stream.start", "start event stream")
-	evtStreamStop  = events.Define("events.stream.stop", "stop event stream")
-)
+var errNoIdentifiers = errors.DefineInvalidArgument("no_identifiers", "no identifiers")
 
 // Stream implements the EventsServer interface.
 func (srv *EventsServer) Stream(req *ttnpb.StreamEventsRequest, stream ttnpb.Events_StreamServer) error {
-	ctx := stream.Context()
-
 	if len(req.Identifiers) == 0 {
-		return nil
+		return errNoIdentifiers
```

Comment:
This is an API behavioral change

Reply:
Doesn't this fall under these two:
> 5. We will not break the API between components and events within minor versions. So at least the same minor versions of components are compatible with each other.
> 6. We reserve the right to fix bugs in API, configuration and storage in patches and minor updates. This may break components, gateways and applications that rely on buggy behavior.

IMO silently failing here is a bug

Reply:
I don't see a bug report that is referenced

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/issues/2569